sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/ErrorMapping.kt

```diff
@@ -2,7 +2,11 @@ package co.nimblehq.sample.compose.ui
 
 import android.content.Context
 import co.nimblehq.sample.compose.R
+import co.nimblehq.sample.compose.domain.exceptions.ApiException
 
 fun Throwable.userReadableMessage(context: Context): String {
-    return context.getString(R.string.error_generic)
+    return when (this) {
+        is ApiException -> error?.message
```

Comment:
@luongvo Should we add the default error message in case of null? 🤔

Reply:
@doannimble Yes, already, could you please re-check that logic again as we're falling back the `error_generic` after a `when` block.

https://github.com/nimblehq/android-templates/blob/58c443a96406175f99cd7abcb2997f3efc4d9f5c/sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/ErrorMapping.kt#L7-L12